### ISSUE TITLE
Issue login to Kodi in new Devices

### ISSUE THREAD

**Issue by ujesh-t** [cguZZman/plugin.googledrive#344]
**Issue login to Kodi in new Devices**

While trying to login using OAuth everything works except google marking the application as dangarous 

**To Reproduce (Required :red_circle:)**

1. Plugin settings > Add Account
2. QR code and login server is displayed
3. Go to the browser and navigate to the URL displayed in Kodi (https://drive-login.herokuapp.com/)
4. Login to Google

**Screenshots**
![Screenshot 2024-05-24 210422](https://github.com/cguZZman/plugin.googledrive/assets/11648010/3a1cd23d-4e40-466f-8367-63247c7fe03f)

**Comment by Gionames** [comment]
Me too. Google Chromecast with Google TV 4K fully updated

**Comment by kamillopl** [comment]
Look at closed issues. This topic was mentioned here: https://github.com/cguZZman/plugin.googledrive/issues/342
TLDR: we have to wait few days/weeks or can use solution created by some user in that issue.

**Comment by yusukeeeeee1234** [comment]
It was resolved, but lately I've been blocked again. 
How do you feel? 
It's been like this for over a week.

**Comment by yusukeeeeee1234** [comment]
I solved it with #342, but the app is blocked again and I can't add an account. Is there any solution?

**Comment by MinhAmazing** [comment]
follow this guide and you can use google drive login normally: https://youtu.be/1piIB5DfuQo?si=tm8hT2okxm6cUDcX

**Comment by M3veronMast3r** [comment]
![Screenshot_20240829-003759](https://github.com/user-attachments/assets/2cad3711-5b03-4724-af8c-8bb41476fde1)
> follow this guide and you can use google drive login normally: https://youtu.be/1piIB5DfuQo?si=tm8hT2okxm6cUDcX

Video is private and not available to public

**Comment by krishbansalkb** [comment]
@MinhAmazing The youtube video is showing as private. Pls share it again

**Comment by Sakuragi90** [comment]
https://github.com/Sakuragi90/kodi-gdrive, check this out for solution ....:)

**Comment by DebankurMandal** [comment]
> https://github.com/Sakuragi90/kodi-gdrive, check this out for solution ....:)

After succesfully following your steps and after successful login, a prompt pops up in Kodi which says "Unable to retrieve the owner information: HTTP Error 403: Forbidden Please try again." Please help us to solve this issue

**Comment by Axanderism** [comment]
Any resolution? Any hint of a fix?
